src/modules/apps/e2e/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { render } from 'ink'
+
+import { apps, tester } from '../../../clients'
+import { RealTimeReport } from './report'
+import { getToken } from '../../../conf'
+import { ManifestEditor } from '../../../lib/manifest/ManifestEditor'
+import { ErrorBoundary } from './errorBoundary'
+import { TestRequest } from '../../../clients/Tester'
+
+
+export default async options => {
+  new EndToEndCommand(options).run()
+}
+
+class EndToEndCommand {
+  options : any
+
+  constructor(options) {
+    this.options = options
+  }
```

Comment:
_nit_: this can be
```suggestion
  constructor(private options) {}
```
Then typescript will automatically assign